src/table/utils/styling-utils.ts

```diff
@@ -311,3 +311,6 @@ export function getSelectionStyle(styling: CellStyle, cellSelectionState: Select
     ...selectionStyling,
   };
 }
+
+export const autoAlign = (cell: EngineAPI.INxCell) =>
```

Comment:
this one returns type of `Align`, I think by returning Align type from this probably you wont need to cast in line 180 of `handle-data.ts` file

but also it might return a default alignment as well because of boolean check